src/collectors/diskhealth/diskhealth.py

```diff
@@ -0,0 +1,150 @@
+# coding=utf-8
+
+"""
+Check Disk Health
+
+Note: The user running Diamond will need to have write access to each mount point
+that should be checked. The test performa a write and a read in each mount.
+
+#### Dependencies
+
+ * /proc/mount
+
+"""
+
+import diamond.convertor
+
+import time
+import os
+import re
+
+from multiprocessing import Pool, TimeoutError
+
+TEST_FILE_SIZE_BYTES = 1048576  # 1MB
+
+
+def _format_metric(device, mount_point, metric):
+    return '%s.%s.%s' % (
+        device.strip('/').replace('/', '_'),
+        'root' if mount_point == '/' else mount_point.strip('/').replace('/', '_'),
```

Comment:
I'd follow the pattern of other collectors here with a configuration element called "raw_stats_only" that, if true, enables the use of "/" characters in the metric name. This will keep disk metrics consistent across collectors.